python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
```

Comment:
I'm mildly uncomfortable with returning a more specific type than the one the underlying Java class guarantees returning - but if @davies or another Python committer agree I'll switch it over.